### PR TITLE
add menu and modify documentation according to the standard documentation

### PR DESCRIPTION
**Pull request by Sibo2rr**
add menu and modify documentation according to the standard documentation

### CODE REVIEW

File: docs/zh_CN/advanced_tutorials/DataAugmentation.md [hunk 1]
```diff
@@ -16,20 +16,20 @@
...
+- [2. 启动命令](#2)
+- [3. 注意事项](#3)
+- [4. 实验结果](#4)
 
 <a name="1"></a>
-## 一、参数配置
+## 1. 参数配置
 
-由于不同的数据增强方式含有不同的超参数，为了便于理解和使用，我们在`configs/DataAugment`里分别列举了8种训练ResNet50的数据增强方式的参数配置文件，用户可以在`tools/run.sh`里直接替换配置文件的路径即可使用。此处分别挑选了图像变换、图像裁剪、图像混叠中的一个示例展示，其他参数配置用户可以自查配置文件。
+由于不同的数据增强方式含有不同的超参数，为了便于理解和使用，我们在 `configs/DataAugment` 里分别列举了  8 种训练 ResNet50 的数据增强方式的参数配置文件，用户可以在 `tools/run.sh` 里直接替换配置文件的路径即可使用。此处分别挑选了图像变换、图像裁剪、图像混叠中的一个示例展示，其他参数配置用户可以自查配置文件。
```
Comment (cuicheng01) [inline docs/zh_CN/advanced_tutorials/DataAugmentation.md]: 8前边空格多了

File: docs/zh_CN/advanced_tutorials/DataAugmentation.md [hunk 2]
```diff
@@ -248,9 +249,10 @@
 ```
 
 <a name="1.10"></a>
-### 1.10 Mixup与Cutmix同时使用
+### 1.10 Mixup 与 Cutmix 同时使用
+
+`Mixup` 与 `Cutmix`同时使用的配置如下，其中用户需要指定额外的参数 `prob`，该参数控制不同数据增强的概率，默认为 `0.5`。
```
Comment (cuicheng01) [inline docs/zh_CN/advanced_tutorials/DataAugmentation.md]: 少了空格

File: docs/zh_CN/advanced_tutorials/knowledge_distillation.md [hunk 3]
```diff
@@ -48,17 +47,17 @@ SSLD 的流程图如下图所示。
 <img src="../../images/distillation/ppcls_distillation.png"  width = "600" />
 </div>
 
-首先，我们从 ImageNet22k 中挖掘出了近400万张图片，同时与 ImageNet-1k 训练集整合在一起，得到了一个新的包含 500 万张图片的数据集。然后，我们将学生模型与教师模型组合成一个新的网络，该网络分别输出学生模型和教师模型的预测分布，与此同时，固定教师模型整个网络的梯度，而学生模型可以做正常的反向传播。最后，我们将两个模型的 logits 经过 softmax 激活函数转换为 soft label ，并将二者的 soft label 做 JS 散度作为损失函数，用于蒸馏模型训练。下面以 MobileNetV3 （该模型直接训练，精度为 75.3%） 的知识蒸馏为例，介绍该方案的核心关键点（ baseline 为 79.12% 的 ResNet50_vd 模型蒸馏 MobileNetV3 ，训练集为 ImageNet1k 训练集， loss 为 cross entropy loss ，迭代轮数为 120epoch ，精度指标为 75.6% ）。
+首先，我们从 ImageNet22k 中挖掘出了近 400 万张图片，同时与 ImageNet-1k 训练集整合在一起，得到了一个新的包含 500 万张图片的数据集。然后，我们将学生模型与教师模型组合成一个新的网络，该网络分别输出学生模型和教师模型的预测分布，与此同时，固定教师模型整个网络的梯度，而学生模型可以做正常的反向传播。最后，我们将两个模型的 logits 经过 softmax 激活函数转换为 soft label ，并将二者的 soft label 做 JS 散度作为损失函数，用于蒸馏模型训练。下面以 MobileNetV3 （该模型直接训练，精度为 75.3%） 的知识蒸馏为例，介绍该方案的核心关键点（baseline 为 79.12% 的 ResNet50_vd 模型蒸馏 MobileNetV3 ，训练集为 ImageNet1k 训练集， loss 为 cross entropy loss ，迭代轮数为 120epoch ，精度指标为 75.6%）。
```
Comment (cuicheng01) [inline docs/zh_CN/advanced_tutorials/knowledge_distillation.md]: 这里很多逗号前有空格，下同，整个文档都有这个问题

File: docs/zh_CN/advanced_tutorials/knowledge_distillation.md [hunk 4]
```diff
@@ -48,17 +47,17 @@ SSLD 的流程图如下图所示。
...
 
-* 改进 loss 计算方法。分类 loss 计算最常用的方法就是 cross entropy loss ，我们经过实验发现，在使用 soft label 进行训练时，相对于 cross entropy loss ， KL div loss 对模型性能提升几乎无帮助，但是使用具有对称特性的 JS div loss 时，在多个蒸馏任务上相比 cross entropy loss 均有 0.2% 左右的收益( `76.0%->76.2%` )， SSLD 中也基于 JS div loss 展开实验。
+* 改进 loss 计算方法。分类 loss 计算最常用的方法就是 cross entropy loss ，我们经过实验发现，在使用 soft label 进行训练时，相对于 cross entropy loss ， KL div loss 对模型性能提升几乎无帮助，但是使用具有对称特性的 JS div loss 时，在多个蒸馏任务上相比 cross entropy loss 均有 0.2% 左右的收益(`76.0%->76.2%`)， SSLD 中也基于 JS div loss 展开实验。
 
-* 更多的迭代轮数。蒸馏的 baseline 实验只迭代了 120 个 epoch 。实验发现，迭代轮数越多，蒸馏效果越好，最终我们迭代了 360epoch ，精度指标可以达到 77.1%(`76.2%->77.1%`) 。
+* 更多的迭代轮数。蒸馏的 baseline 实验只迭代了 120 个 epoch 。实验发现，迭代轮数越多，蒸馏效果越好，最终我们迭代了 360 epoch ，精度指标可以达到 77.1%(`76.2%->77.1%`) 。
 
-* 无需数据集的真值标签，很容易扩展训练集。 SSLD 的 loss 在计算过程中，仅涉及到教师和学生模型对于相同图片的处理结果（经过 softmax 激活函数处理之后的 soft label ），因此即使图片数据不包含真值标签，也可以用来进行训练并提升模型性能。该蒸馏方案的无标签蒸馏策略也大大提升了学生模型的性能上限（ `77.1%->78.5%` ）。
+* 无需数据集的真值标签，很容易扩展训练集。 SSLD 的 loss 在计算过程中，仅涉及到教师和学生模型对于相同图片的处理结果（经过 softmax 激活函数处理之后的 soft label ），因此即使图片数据不包含真值标签，也可以用来进行训练并提升模型性能。该蒸馏方案的无标签蒸馏策略也大大提升了学生模型的性能上限( `77.1%->78.5%`)。
```
Comment (cuicheng01) [inline docs/zh_CN/advanced_tutorials/knowledge_distillation.md]: 括号前后不加空格，下同

**Review by cuicheng01 — DISMISSED** [review]

**Review by cuicheng01 — APPROVED** [review]
